### PR TITLE
Solana/about pages

### PR DESCRIPTION
**Pull request by XiaoYhun**

### CODE REVIEW

File: src/constants/networks.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ export const NETWORKS_INFO_CONFIG: NETWORKS_INFO_CONFIG_TYPE = {
   [ChainId.OASIS]: oasis,
   [ChainId.OPTIMISM]: optimism,
   [ChainId.SOLANA]: solana,
+  [ChainId.SOLANA]: solana,
```
Comment (namgold) [inline src/constants/networks.ts:59]: duplicated?

File: src/pages/About/AboutKyberSwap.tsx [hunk 2]
```diff
@@ -171,6 +175,8 @@ export const KSStatistic = () => {
 }
 
 function AboutKyberSwap() {
+  const { chainId } = useActiveWeb3React()
+  const isSolana = chainId === ChainId.SOLANA
```
Comment (namgold) [inline src/pages/About/AboutKyberSwap.tsx:179]: use isSolana `from 'constants/networks'`

File: src/pages/About/KyberSwapGeneralIntro.tsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ import { MoneyBagOutline } from 'components/Icons'
 import useMixpanel, { MIXPANEL_TYPE } from 'hooks/useMixpanel'
 import useTheme from 'hooks/useTheme'
 
-const KyberSwapGeneralIntro = () => {
+const KyberSwapGeneralIntro = ({ isSolana }: { isSolana: boolean }) => {
```
Comment (namgold) [inline src/pages/About/KyberSwapGeneralIntro.tsx:12]: I think it will be better to not pass it as props, but make it call useActiveWeb3 directly

Comment (namgold) [reply src/pages/About/KyberSwapGeneralIntro.tsx:12]: to reduce dependancy between it and its parent

File: src/pages/About/styleds.tsx [hunk 4]
```diff
@@ -220,29 +220,35 @@ export const FooterContainer = styled.div`
...
+      width: calc(100% / 3 - 52px);
+    }
   `}
 
   ${({ theme }) => theme.mediaWidth.upToExtraSmall`
-    padding: 12px;
-    row-gap: 24px;
-    grid-template-columns: repeat(2, minmax(0, 1fr));
+    & > * {
+      width: calc(50% - 52px);
```
[the remark this answers is not in the document]
Comment (XiaoYhun) [reply src/pages/About/styleds.tsx:250]: its normal, like gap value or padding value

File: src/pages/About/KyberSwapGeneralIntro.tsx (hunk 3 above)
Comment (XiaoYhun) [reply src/pages/About/KyberSwapGeneralIntro.tsx:12]: in many cases, pass prop is not bad if only pass 1 layer, more than 1 will become prop drilling. Use hook in other hand make code look duplicated, more function calls. 
In this case, KyberSwapGeneralIntro  only a simple render component, its ok to take a prop from parent. Use hook mean should run isSolana function one more time.

**Pull request merged**